moxie-protocol/src/utils.ts

```diff
@@ -56,8 +60,18 @@ export function getOrCreatePortfolio(userAddress: Address, subjectAddress: Addre
   return portfolio
 }
 
-export function savePortfolio(portfolio: Portfolio, block: ethereum.Block): void {
+export function savePortfolio(portfolio: Portfolio, block: ethereum.Block, allowDelete:bool = false): void {
```

Comment:
Please add documentation to this method, explaining `allowDelete` argument & unique holder logic

Reply:
improved the implementation, we can avoid using allowDelete flag